template/README.md

```diff
@@ -182,20 +203,39 @@ vagrant ssh -c 'vault read consul/creds/dev-team'
 vagrant ssh -c 'vault read consul/creds/admin-team'
 ```
 
-*Tokens can be used to access UI (different access level depends on role)
+> :bulb: Tokens can be used to access UI (different access level depends on role)
+
+
+## Usage
+### Commands
+There are several commands that help to run the vagrant-box:
+- `make install` installs all prerequisites
```

Comment:
I would suggest notifying the user, that he/she does not need to run it every time.
```suggestion
- `make install` installs all prerequisites. Run once.
```